python/tests/test_ephemeris.py

```diff
@@ -186,3 +186,48 @@ def test_parameters():
   valid = 1
   fit_interval = 4
   assert not e.Ephemeris.is_params_valid(valid, fit_interval, toe, time)
+
+def test_glo_sat_state():
+  eph1 = e.Ephemeris(**{'sid': {'sat': 1, 'code': 3,},
+                     'toe': {'wn': 1884, 'tow': 260100.0,}, #from GPS
+                     'ura': 2.0,
+                     'fit_interval': 4 * 60 * 60,
+                     'valid': 1,
+                     'healthy': 1,
+                     'glo': {  #RINEX data at 2016-02-17T00:15:00
+                              'gamma' : 0,
+                              'tau' : 0.000145863741636,
+                              'pos' : [15401788.5742,20181713.8672,2390948.73047],
+                              'vel' : [-106.175422668,-338.997840881,3540.23456574],
+                              'acc' : [-9.31322574616e-07,0,-9.31322574616e-07],
+                            }})
+  eph2 = e.Ephemeris(**{'sid': {'sat': 1, 'code': 3,},
+                     'toe': {'wn': 1884, 'tow': 261900.0,}, #from GPS
+                     'ura': 2.0,
+                     'fit_interval': 4 * 60 * 60,
+                     'valid': 1,
+                     'healthy': 1,
+                     'glo': {  #RINEX data at 2016-02-17T00:45:00
+                              'gamma' : 0,
+                              'tau' : 0.000145862810314,
+                              'pos' : [14620682.1289,19044145.9961,8588398.92578],
+                              'vel' : [-781.462669373,-888.925552368,3301.05400085],
+                              'acc' : [0,0,-9.31322574616e-07],
+                            }})
+  #check that both ephemeris are valid at 2016-02-17T00:30:00
+  assert eph1.is_valid(t.GpsTime(**{ 'wn': 1884, 'tow': 261000.0,}))
+  assert eph2.is_valid(t.GpsTime(**{ 'wn': 1884, 'tow': 261000.0,}))
+  assert eph1.is_healthy()
+  assert eph2.is_healthy()
+  #set time in between 2 ephemeris
+  time = t.GpsTime(**{ 'wn': 1884, 'tow': 261000.0,})
+  #calculate SV orbits at the time
+  pos1, vel1, clock_err1, clock_rate_err1 =  eph1.calc_sat_state(time)
+  pos2, vel2, clock_err2, clock_rate_err2 =  eph2.calc_sat_state(time)
+  clock_rate_err1-clock_rate_err2
+  #check if position difference of the SV in 2.5 m range
+  assert np.allclose(pos1, pos2, 0, 2.5)
+  #check if velosity difference of the SV in 0.005 m/s range
+  assert np.allclose(vel1, vel2, 0, 0.005)
+  assert abs(clock_err1 - clock_err2) < 1.0e-9
+  assert abs(clock_rate_err1 - clock_rate_err2) < 1.0e-9
```

Comment:
Could you please clarify how reference position and velocity were calculated at lines 52-53? If I use real data from ephemeris I'll have huge difference, because ephemeris keeps unchanged within 30 mins.